domains/shin.json

```diff
@@ -7,6 +7,8 @@
     "twitter": "tachib_shin"
   },
   "record": {
-    "CNAME": "tachibana-shin.github.io"
+    "CNAME": "tachibana-shin.github.io",
```

Comment:
```suggestion
    "A": [
        "185.199.108.153",
        "185.199.109.153",
        "185.199.110.153",
        "185.199.111.153"
    ],
```

Reply:
This will resolve the no CNAME with other records issue and your website will still display.

Reply:
> This will resolve the no CNAME with other records issue and your website will still display.

thank you for the fixes to my pull request